package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
```

Comment:
why is there two package.json files in the repo? Since this is the new added package.json it probably needs to be removed. When running npm install or yarn add (whatever you are using) make sure you are in the correct directory!

Reply:
noted